tour/batch-scanner.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Batch Scanner
 ---
-Running on a single thread, a Scanner will retrieve a single Range of data and return Keys in sorted order. A [BatchScanner] 
-will retrieve multiple Ranges of data using multiple threads.  A BatchScanner can be more efficient but does not guarantee Keys will be returned in sorted order.
+Running on a single thread, a Scanner will retrieve a single Range of data and return keys in sorted order. A [BatchScanner] 
```

Comment:
This is another one where we literally mean objects of type `Key`, so uppercase is okay. I could go either way on these things, like `String` and `Key`, since their functional name and their proper name are the same and interchangeable. It might be nice if we were a bit more consistent on case, and syntax highlighting (see my other comment regarding `String` for the suggestion to use syntax highlighting).

Reply:
I see your point. At the time of committing I was focused on grammar but didn't take into consideration that we were referring to objects. I'll apply the fixes first thing tomorrow morning.